### PR TITLE
Use simple PR rather than 2/3 vote to graduate projects from incubation

### LINKED ISSUE #567: Incubation process clarifications
As with a few governance and process documents that were written in a vacuum, there needs to be some clarification around Incubation process. The documentation states that the incubation graduation process concludes with a vote, but this step seems cumbersome and unnecessary.

https://github.com/openjs-foundation/cross-project-council/blob/master/PROJECT_PROGRESSION.md

The consensus at the most recent CPC meeting is that we should use the consensus seeking model at both entry and exit of incubation process:
- during application silent period, CPC members have an opportunity to object
- incubation process happens, onboarding checklist, charter approval, etc
- at graduation there is a final notification that project is graduating, this is an opportunity for any last objections but none should be expected as the process has been ongoing

For this last step, I propose that this happens as follows:
1. A pull request is opened to add the project to the readme at the appropriate stage (Impact, Growth, etc)
2. This pull request links to appropriate onboarding documentations, including a link to the onboarding checklist completed as well as charter issue approved
3. This is an opportunity to object, but none should be expected.
4. This pull request would follow process as any other general pull request: 2 approvals, open 72 hrs, no objections (https://github.com/openjs-foundation/cross-project-council/blob/master/GOVERNANCE.md#merging-prs-into-this-repository)

### PR DESCRIPTION
**Pull request by eemeli**
This resolves #567, where it was discussed that the current requirement for a 2/3 vote of the CPC to graduate a project from incubation is superfluous.

As related cleanup, this also drops the duplicate and currently out-of-date `PROEJCTS.md` file; the up-to-date list of projects is in `README.md`, and we don't really have a reason to duplicate that. I also removed the "Graduation Criteria" section of the "Incubation Stage" description in `PROJECT_PROGRESSION.md`, as it too duplicates with slightly different wording text from the application process.

### CODE REVIEW

File: PROJECT_PROGRESSION.md [hunk 1]
```diff
@@ -100,7 +101,7 @@ This is an informational checklist to help projects onboard into the OpenJS Foun
 - [ ] Publish Project Charter on website or GitHub
 - [ ] Update legal copyright notice on project website and GitHub
 - [ ] Add OpenJS Foundation logo to project website
-- [ ] Add Project logo to OpenJS Foundation website; update PROJECTS.md file
+- [ ] Add Project logo to OpenJS Foundation website
```
Comment (mcollina) [inline PROJECT_PROGRESSION.md:104]: This should be "update the README" instead.

**Comment by eemeli** [comment]
Rebased to drop the line-ending-whitespace cleanup. In case anyone else is using VS Code, this is the config that'll prevent it from automatically "fixing" it:
```js
{
  "[markdown]": { "files.trimTrailingWhitespace": false }
}
```

File: PROJECT_PROGRESSION.md (hunk 1 above)
Comment (eemeli) [reply PROJECT_PROGRESSION.md:104]: Added a separate action item to the checklist clarifying this.

File: PROJECT_PROGRESSION.md [hunk 2]
```diff
@@ -65,7 +65,8 @@ The Application Champion is a member of the CPC who commits to do the following:
         (ex. perhaps a different foundation would be a more suitable endeavor.)
    * Discussion/agreement between applicant, application manager and Foundation media team to agree on announce date.
 1. Project goes through process of adhering to [on-boarding checklist](#onboarding-checklist).
-1. Project presents to CPC for final acceptance via 2/3 supermajority vote and appropriate initial stage is determined.
+1. Project files an issue in this repository requesting its charter to be reviewed by the CPC
```
Comment (joesepi) [inline PROJECT_PROGRESSION.md:68]:
```suggestion
1. Project completes a charter review process by the CPC as outlined in the [CPC Governance](https://github.com/openjs-foundation/cross-project-council/blob/master/GOVERNANCE.md#approving-project-charters). NOTE: this process takes two weeks to complete
```

File: PROJECT_PROGRESSION.md [hunk 3]
```diff
@@ -65,7 +65,8 @@ The Application Champion is a member of the CPC who commits to do the following:
...
+1. Project graduates from incubation by a pull request closing the charter review issue and adding it to the README.md file at its appropriate initial stage.
```
Comment (joesepi) [inline PROJECT_PROGRESSION.md:69]:
```suggestion
1. Project graduates from incubation by a pull request adding the project to the README.md file at its appropriate initial stage. The merging of this pull request proceeds as any other pull request to the README.md file.
```

**Comment by eemeli** [comment]
Applied changes suggested by @joesepi, hence re-requesting earlier reviews.

**Comment by mhdawson** [comment]
Tagged with "waiting on the board" as I think we should at least give an FYI to see if there are any concerns.  

@joesepi let me know once you think it has enough CPC approvals and then I'll send that FYI on.

**Comment by joesepi** [comment]
@mhdawson I think we have adequate approvals here. Please send FYI to board and report back when you think we are good to move this forward. Thanks!

**Comment by kborchers** [comment]
@joesepi We already informed them that this change was coming in the last board meeting and there were no objections or concerns so I believe we are good to move forward.

**Pull request merged**